core/complexStructures/processedMiniBlocks.go

```diff
@@ -0,0 +1,132 @@
+package complexStructures
+
+import (
+	"sync"
+
+	"github.com/ElrondNetwork/elrond-go/logger"
+	"github.com/ElrondNetwork/elrond-go/process/block/bootstrapStorage"
+)
+
+var log = logger.GetOrCreate("core/complexStructures")
+
+// ProcessedMiniBlocks is used to store all processed min blocks hashes for e meta block hash
+type ProcessedMiniBlocks struct {
```

Comment:
Please rename it to something like `ProcessedMiniBlockTracker`

Reply:
done.